test/common/TestCommonEqEpsilon.hs

```diff
@@ -5,14 +5,41 @@ module TestCommonEqEpsilon (EqEpsilon, setEpsilonEq, assertCloseElem, (@?~)) whe
 import Prelude
 import Data.Typeable
 
+import           Control.Monad (unless)
 import           Data.IORef
 import qualified Data.Vector.Generic as VG
 import qualified Data.Vector.Storable as VS
 import           System.IO.Unsafe
-import qualified Test.HUnit.Approx
 import           Test.Tasty.HUnit
 import           Test.Tasty.Options
 
+-- | Copied from: https://hackage.haskell.org/package/HUnit-approx-1.1.1.1/docs/src/Test-HUnit-Approx.html#assertApproxEqual
+-- because we do not want to depend on Test.HUnit.Approx . Instead, we're using an outdated copy of HUnit hardcoded into
+-- Tasty (Test.Tasty.HUnit).
+--
+-- Copyright (c) 2014, Richard Eisenberg
```

Comment:
Hi @goldfirere , I have stolen some of your code. Hope you're OK with that?... This is to get rid of dependency on `HUnit-approx` (`Tasty` contains its own copy of `HUnit` and if we have both, it leads to problems).